docs/source/help_and_reference/roadmap.rst

```diff
@@ -52,81 +49,51 @@ affine image registration
-can add GPU-enabled methods to starfish. Finally, we will create benchmarks to track the speed of starfish operations
-to catch performance regressions.
-
-Resolving overlaps between fields of view
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-Work to date has focused on processing individual fields of view, with functional but limited
-support for combining results across fields of view. We plan to improve the ability to merge
-features, such as spots or cells, that overlap multiple fields of view.
-
-Durability & Versioning `#1309 <https://github.com/spacetx/starfish/issues/1309>`_
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-There are a number of areas where we seek to improve the durability of the package for users. These include improving
-logging and versioning file formats, so that it is always clear how a given output was produced and how to reproduce it.
-
-Quality Control `#61 <https://github.com/spacetx/starfish/issues/61>`_
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-We’ve begun to explore assay-agnostic quality control metrics that can be used to assess the quality of an experiment.
-We plan to add support for these metrics so our users are better equippied to evaluate the quality of their data and
-data processing routines.
-
-Simplifying Contribution Patterns `#1521 <https://github.com/spacetx/starfish/issues/1521>`_
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-We are very interested in contribution of algorithms in active research areas, such as segmentation, or spot decoding.
-Therefore, we want to make starfish easy and fun to contribute to. We recognize that parts of starfish, by virtue of the
-multi-field of view data flow, are quite complex. We will endeavor to simplify the package and clarify API documentation
-to streamline future contribution. We've opened an issue to source points of confusion. Comments on this issue will help
-us identify places where our development team can focus our efforts.
-
-Proteomics Support
-~~~~~~~~~~~~~~~~~~
-Proteomics assays follow similar data analysis patterns to RNA-based assays, but have some unique requirements that we
-expect to require additional algorithms. Starfish would like to add support for these assays.
+
+Support for proteomics assays `#1674 <https://github.com/spacetx/starfish/issues/1674>`_
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Proteomics assays like CODEX, CyCIF, MIBI, IMC, and others follow similar image processing patterns
+to RNA-based assays, but have some unique requirements that we expect to require additional algorithms
+and components in starfish. We think adding support for these assays and hybrid transcriptomics/proteomics
+assays would be valuable and would welcome community contributions.
+
+Support for segmentation-free cell assignment `#1675 <https://github.com/spacetx/starfish/issues/1675>`_
```

Comment:
Awesome!

Reply:
To be clear... we're not planning to do this ourselves, but would welcome contributions :D